tfhe/src/integer/gpu/ciphertext/compressed_ciphertext_list.rs

```diff
@@ -0,0 +1,207 @@
+use crate::core_crypto::gpu::glwe_ciphertext_list::CudaGlweCiphertextList;
+use crate::core_crypto::gpu::CudaStreams;
+use crate::core_crypto::prelude::CiphertextModulusLog;
+use crate::integer::ciphertext::DataKind;
+use crate::integer::gpu::ciphertext::info::CudaBlockInfo;
+use crate::integer::gpu::ciphertext::{
+    CudaRadixCiphertext, CudaSignedRadixCiphertext, CudaUnsignedRadixCiphertext,
+};
+use crate::integer::gpu::list_compression::server_keys::{
+    CudaCompressionKey, CudaDecompressionKey,
+};
+
+pub struct CudaCompressedCiphertextList {
+    pub(crate) packed_list: (
+        CudaGlweCiphertextList<u64>,
+        Vec<CudaBlockInfo>,
+        CiphertextModulusLog,
+    ),
+    info: Vec<DataKind>,
+}
+impl CudaCompressedCiphertextList {
+    pub fn len(&self) -> usize {
+        self.info.len()
+    }
+
+    pub fn is_empty(&self) -> bool {
+        self.info.len() == 0
+    }
+
+    pub fn get(
+        &self,
+        index: usize,
+        decomp_key: &CudaDecompressionKey,
+        streams: &CudaStreams,
+    ) -> CudaRadixCiphertext
+where {
```

Comment:
where clause should be removed here it's empty

Reply:
Done!